lib/active_admin/application.rb

```diff
@@ -117,6 +117,8 @@ def initialize
 
     # Active Admin makes educated guesses when displaying objects, this is
     # the list of methods it tries calling in order
+    # Note that Formtastic has also have `collection_label_methods` similar to this
```

Comment:
Note that Formtastic also has 'collection_label_methods' similar to this
used by auto generated dropdowns in filter or belongs_to field of Active Admin